apps/vr-tests/src/stories/Button.stories.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import Screener, { Steps } from 'screener-storybook/src/screener';
 import { storiesOf } from '@storybook/react';
-import { Button } from '@fluentui/react-button';
+import { Button } from '@fluentui/react';
 import { AddIcon } from '@fluentui/react-icons-mdl2';
 import { TeamsTheme } from '@fluentui/storybook/lib/themes/v8/index';
 import { withThemeProvider } from '@fluentui/storybook';
 import { FabricDecorator, FabricDecoratorFullWidth } from '../utilities/index';
 
-storiesOf('Button Next', module)
```

Comment:
Could you similarly remove the `Next` moniker in `CompoundButton`, `MenuButton`, `SplitButton` and `ToggleButton` stories as well since you're already doing it here?

Reply:
We also probably want to check the rest of this file for usage of `Next` since I think there were multiple stories in each one.

Reply:
done.